readme.md

```diff
@@ -0,0 +1,27 @@
+# Housing Needs Single View API
+
+## Getting Local Dev Environment Started
+
+Run npm install in the following directories:
+- / (root)
+- api/
+- authorizer/
+
+Set up the environment variables listed in [.env.sample](https://github.com/LBHackney-IT/housing-needs-single-view-proto-api/blob/master/.env.sample) and save them in a `.env` file, the relevant values could be found in the AWS param store
+
+Add the following to the hosts file
+```
+127.0.0.1 localdev.hackney.gov.uk
+```
+
+Set up the DB by running
+```
+$ docker-compose up db
+```
+
+Start the API by running
+```
+$ npm run start
+```
+
+Play with the API by visiting http://localdev.hackney.gov.uk:3000/customers?firstName=john&lastName=smith
```

Comment:
Probably worth adding a note to say that you need to log in with a Hackney email address with this method. Otherwise you can pass --noAuth to the sls command to run it without auth